models/bamf_nnunet_ct_liver/utils/BamfProcessorRunner.py

```diff
@@ -0,0 +1,79 @@
+"""
+-------------------------------------------------
+MedicalHub - Run Module for ensembling nnUNet inference.
```

Comment:
I just noticed we still have the very old headers here.
Can we replace `MedicalHub` with `MHub` for consistent branding :)?